bin/yaml/libraries.yaml

```diff
@@ -2722,6 +2722,35 @@ libraries:
       - 1.16.0
       - 1.17.0
       type: github
+  kokkos:
+      build_type: cmake
+      lib_type: static
+      make_targets:
+      - all
+      extra_cmake_arg:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DKokkos_ENABLE_CUDA=ON
+      - -DKokkos_ARCH_TURING75=ON
```

Comment:
Is that the GPUs that is being deployed?

Reply:
@partouf will this "just work" do you think?

Reply:
We'll need a new way of building libraries for GPU/Cuda similar to Windows, so lots of if's and things, but this is useful information either way. Will probably need rename to something like `gpu_cmake_arg`